well-known-bots.json

```diff
@@ -25,6 +34,7 @@
       "search-engine"
     ],
     "pattern": "Googlebot-Mobile",
+    "verification": [],
```

Comment:
Won't this be verified the same as Googlebot?

Reply:
They don't mention this one on any of their three lists of crawlers ([common](https://developers.google.com/search/docs/crawling-indexing/google-common-crawlers), [special case](https://developers.google.com/search/docs/crawling-indexing/google-special-case-crawlers), [user triggered](https://developers.google.com/search/docs/crawling-indexing/google-user-triggered-fetchers)). I only added verification details for bots that they document. Given that they haven't included it its possible that it may no longer be active.

Reply:
Please open an issue to look into this. We will be able to search requests arriving to see if it exists.

Reply:
https://github.com/arcjet/well-known-bots/issues/23